### DISCUSSION TITLE
Control the inside speaker volume of monitor?

### DISCUSSION THREAD

**Discussion by WangWH0101** [xanderfrangos/twinkle-tray#514] [General]
Many external monitors have inside speakers now, some of which on portable monitors are actually quite useful. I'm not quite sure but it seems that the DDC/CI functions supports not only the brightness but also the hardware-level volume change of inside speakers, which has to be changed through the physical menu bar. Therefore, I'm wondering whether it's possible to control the volume of inside speakers with similar method based on DDC/CI functions? ----Solved! Just remember to check the DDC/CI features in the menu.

**Comment by xanderfrangos** [comment]
Hi @WangWH0101,

If your monitor supports volume over DDC/CI, it will display here:

![Screenshot 2022-12-05 091840](https://user-images.githubusercontent.com/33106561/205659475-500d6f9e-b302-41af-9277-c2d8641d39d5.png)

You can enable the Volume function and modify it from Twinkle Tray.

If it doesn't show up, it isn't supported.

I hope that helps!

**Reply by WangWH0101** [reply]
Thank you so much! I think I've forgot to check that setting option:)
